symbolic-symcache/tests/test_writer.rs

```diff
@@ -127,6 +127,25 @@ fn test_write_functions_macos() -> Result<(), Error> {
     Ok(())
 }
 
+// Tests that functions with identical names, compilation directories, and languages but different
+// entry_pcs have separate, distinct entries in the symcache. The specific use case generating this
+// is two identically-named static C functions nestled in two different files sharing a common
+// compilation directory.
```

Comment:
would be nice to also commit the source files, and instructions to reproduce the fixture (as in: compile flags, etc)

Reply:
done 🎉  it's an aggressively simple example so the build process isn't too complex, thankfully. cross-compiling might be another problem, but i'm just going to close my eyes to that problem for the time being.